library/std/src/os/wasi/ffi.rs

```diff
@@ -1,7 +1,6 @@
 //! WASI-specific extensions to primitives in the [`std::ffi`] module
 //!
 //! [`std::ffi`]: crate::ffi
-
 #![stable(feature = "rust1", since = "1.0.0")]
 
```

Comment:
Everything in `library/std/src/os/wasi/` looks like it got a line removed but the file is otherwise untouched. Assume this just happened because of fixups, could you reset these? Just look at the diff.

I think a space between inner docs and inner attributes is common anyway (@workingjubilee could certainly say for sure).

Reply:
Common, not required? Certainly better to not diff the file if nothing actually changed otherwise.

Reply:
I was thinking more about `library/std/src/os/wasip2/mod.rs` that both had an update and removed the line. But yeah, I'm mildly surprised rustfmt doesn't do something here.